### ISSUE TITLE
No function clause matching when running mix test on a phoenix app

### ISSUE THREAD

**Issue by MiyamotoAkira** [livehelpnow/tds_ecto#19]
Hi,

I get the following error when trying to run mix test on a phoenix app:
   *\* (FunctionClauseError) no function clause matching in Tds.Connection.handle_info/2  
   *\* (FunctionClauseError) no function clause matching in Tds.Connection.handle_info/2  
       (tds) lib/tds/connection.ex:173: Tds.Connection.handle_info({:udp_error, #Port<0.9726>, :econnreset},

I have checked the file connection.ex, and doesn't seem to be a handle_info function.

I'm running Tds_Ecto 1.0.0 out of Hex. Tds version is 0.5.1. And phoenix_ecto is 1.2.0

I have not run ecto.create nor ecto.migrate, as the database does already exists. I have used mix phoenix.gen.json to create all the necessary files.

**Comment by mobileoverlord** [comment]
I think I might have to handle error messaging better, but it is having trouble connecting to the sql server browser to look up the server port. Could you provide the config you are using?

**Comment by MiyamotoAkira** [comment]
With the actual values changed, I can.

``` elixir
config :rest_api, RestApi.Repo,
  adapter: Tds.Ecto,
  username: "myusername",
  password: "mypassword",
  database: "mydatabase",
  hostname: "mymachine",
  instance: "myinstance"
```

The actual values are used on an ADO.NET connection on C#, so I know that the connection does work.

**Comment by mobileoverlord** [comment]
What version of Sql Server are you running?
Also, is SQL Server Browser running on your SQL Box

**Comment by MiyamotoAkira** [comment]
Is SQL Server 2008R2. Yes, SQL Browser is running on the machine

**Comment by mobileoverlord** [comment]
I just tested a brand new Phoenix 1.0.2 app using tds_ecto 1.0.0 connecting to an instance on a SQL 2008 R2 DB and it worked fine, I was able to query the db. Being that this is a UDP error I would check network routing and ensure that the port is accessible and open from where you are. I have committed a change to tds master which will clarify the error you received. Let me know if you are able to get further on this.
